gapic/schema/wrappers.py

```diff
@@ -737,6 +738,27 @@ def field_headers(self) -> Sequence[str]:
 
         return next((tuple(pattern.findall(verb)) for verb in potential_verbs if verb), ())
 
+    @property
+    def http_opt(self) -> Optional[Dict[str, str]]:
+        """Return the http option for this method."""
+        http = self.options.Extensions[annotations_pb2.http].ListFields()
+        if len(http) < 1:
+            return None
+
+        http_method = http[0]
+        answer: Dict[str, str] = {
+            'method': http_method[0].name,
+            'url': http_method[1],
```

Comment:
Consider having "template" in the name: maybe `'path_template'` would be more descriptive and accurate.